historical_system_profiles/config.py

```diff
@@ -14,3 +14,10 @@
 bootstrap_servers = os.getenv("BOOTSTRAP_SERVERS", "kafka:29092").split(",")
 consume_topic = os.getenv("CONSUME_TOPIC", None)
 group_id = os.getenv("GROUP_ID", None)
+listener_mode = os.getenv("LISTENER_MODE", "REST_SERVICE")
```

Comment:
I would find `service_mode` easier to grok as I don't immediately think that this is related to the listener portion of the code.

Reply:
This might make the code slightly more complex,  but in addition I would find it easier to understand if there was:

service_mode: listener service or api service
listener_type: deleter or archiver

What will the names of the deployments be?

Reply:
Updated per comment.

Right now the names are "system-profile-archiver", "system-profile-deleter" and "historical-system-profile-backend". I'll probably change the first two to "historical-system-profile-archiver" and "historical-system-profile-deleter".